src/lib.rs

```diff
@@ -686,6 +758,7 @@ struct Params {
     from: Option<syn::Path>,
     bits: usize,
     debug: bool,
+    defmt: bool,
```

Comment:
If we only have the `debug` attribue, an enum (similar to the field `Order`) would be better.